mptests/write_viam_name_test.go

```diff
@@ -1,4 +1,4 @@
-package arm
+package mptests
```

Comment:
None of the tests in this package are meaningful anymore.  Performance testing is done better in a lot of other places and this specific test is not actually testing anything in this arm module.  My preference would be to remove them.

If we have tests they should be for code owned by this repo

Reply:
agree - but was going to let peter move when he's ready